x-pack/plugins/security_solution/public/management/links.test.ts

```diff
@@ -130,6 +130,7 @@ describe('links', () => {
       (calculateEndpointAuthz as jest.Mock).mockReturnValue({
         canIsolateHost: false,
         canUnIsolateHost: true,
+        canAccessEndpointManagement: true,
```

Comment:
Why is this needed now?

Reply:
In order to ensure that we're testing as a `superuser`.